src/reason_pprint_ast.ml

```diff
@@ -1169,17 +1169,25 @@ let isListy = function
 let strip_trailing_whitespace str =
    Str.global_replace (Str.regexp " +$") "" str
 
-let easyFormatToFormatter f x =
+
+let easyFormatToFormatterImpl f x =
   let buf = Buffer.create 1000 in
   let fauxmatter = Format.formatter_of_buffer buf in
   let _ = Format.pp_set_margin fauxmatter settings.width in
   if debugWithHtml.contents then
     Easy_format.Pretty.define_styles fauxmatter html_escape html_style;
   let _ = Easy_format.Pretty.to_formatter fauxmatter x in
   let trimmed = strip_trailing_whitespace (Buffer.contents buf) |> String.trim in
-  Format.fprintf f "%s\n" trimmed;
+  Format.fprintf f "%s\n" trimmed
+
+let easyFormatToFormatter f x =
+  easyFormatToFormatterImpl f x;
   pp_print_flush f ()
 
+let easyFormatToStringFormatter x =
+  ignore (flush_str_formatter ());
```

Comment:
A safety thing - it clears the formatter before writing to it incase anything was added earlier.